### PR TITLE
Create ngs_annotations.json

### PR DESCRIPTION
**Pull request by amapeters**
NGS assay annotations

### CODE REVIEW

**Comment by amapeters** [comment]
I thought we wanted to split the assays up? Leave assay in minified, and assay specific terms in separate schema. What about assay specific platforms?

**Comment by sgosline** [comment]
The assay-specific _keys_ should be split up - the term 'assay' should only be defined in one file, and I believe this is still a minified.

**Comment by sgosline** [comment]
Also, can you please move the platform to this file too? I feel like that's ngs specific, no?

**Comment by amapeters** [comment]
Yes, but then we end up with the platform key in multiple files

**Comment by sgosline** [comment]
Aside from confocalImaging (which can arguably be put somewhere else) everything is either array or sequencing - perhaps we can call it arrayPlatform and sequencePlatform and put them in the respective tech-specific dictionaries?

**Comment by amapeters** [comment]
we have already stared to use the key 'platform' - can edit if all agree we should, but we need to lock this down soon

File: ngs_annotations.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+    "assay": [
```
Comment (kdaily) [inline ngs_annotations.json]: assay stays in minified

File: ngs_annotations.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        false 
+       
+    ],
+    "libraryPrep": [
+        "rRNAdepletion",
+        "polyAselection",
+        "lncRNAenrichment",
+        "miRNAenrichment"
+    ],
+    "fileSubFormat": [
```
Comment (kdaily) [inline ngs_annotations.json]: goes to neuro (STAR aligner specific)

File: ngs_annotations.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        "rRNAdepletion",
+        "polyAselection",
+        "lncRNAenrichment",
+        "miRNAenrichment"
+    ],
+    "fileSubFormat": [ 
+        "TranscriptomeSAM",
+        "SortedByCoordinate"
+    ],
+    "libraryPrep": [
```
Comment (kdaily) [inline ngs_annotations.json]: delete this one

**Comment by kdaily** [comment]
Review me please, @kkdang!

**Comment by kdaily** [comment]
Or @sgosline!

**Pull request merged**